lib/hyrax/transactions/steps/save_access_control.rb

```diff
@@ -18,8 +18,8 @@ class SaveAccessControl
         # @return [Dry::Monads::Result]
         def call(obj)
           return Success(obj) unless obj.respond_to?(:permission_manager)
-          obj.permission_manager&.acl&.save ||
-            (return Failure[:failed_to_save_acl, acl])
+          acl = obj.permission_manager&.acl
+          acl&.save || (return Failure[:failed_to_save_acl, acl])
```

Comment:
I noticed this missing definition of `acl` on line 22 when writing the `delete_access_control` step so I refactored it to match.

Reply:
👍🏻